lib/unleash/configuration.rb

```diff
@@ -93,6 +96,18 @@ def use_bootstrap?
       self.bootstrap_config&.valid?
     end
 
+    def streaming_mode?
+      return false if RUBY_ENGINE == 'jruby'
```

Comment:
I think we should consider raising an exception here or logging an error. I would be very annoyed if an SDK I used didn't do what I told it and quietly did something different

Reply:
fair point